cloudsync/provider.py

```diff
@@ -245,6 +245,8 @@ def authenticate(self) -> Creds:
                 log.error("oauth error %s", repr(e))
                 self.disconnect()
                 raise CloudTokenError(repr(e))
+            finally:
+                self._oauth_config.server_close()
```

Comment:
Furthering the earlier comment about `server_close`, is the server stopped at this point?

Reply:
it is shutdown if wait_auth() returns / there are no exceptions.

if there is an exception, we are no worse off than before.

Reply:
moved discussion to VFM-10044